site/_posts/2020-09-23-jerry-hopper-fusionauth.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog-post
+title: Jerry loved FusionAuth so much he built an ARM image for it
```

Comment:
Jerry riffs on FusionAuth GDPR compliance and that time he built an ARM image for it